src/simulator_worker/utils.py

```diff
@@ -132,9 +155,7 @@ def create_output_esdl(
     influxdb_profile_manager.save_influxdb(
         measurement=input_uuid,
         field_names=influxdb_profile_manager.profile_header[1:],
-        tags={
-            "output_esdl_id": esh.energy_system.id
-        },
+        tags={"output_esdl_id": esh.energy_system.id},
```

Comment:
I believe the energy_system_id is currently unchanged between the input & output ESDL's? If so, the output ESDL needs to have a different (uu)id as the ESDL itself has been changed.

Reply:
the energy_system_id is set to a new UUID at the start of create_output_esdl() on line 125

Reply:
Cheers!